src/olympia/reviewers/admin.py

```diff
@@ -29,5 +29,11 @@ class ReviewerScoreAdmin(admin.ModelAdmin):
     list_filter = ('note_key',)
 
 
+class ReviewActionReasonAdmin(admin.ModelAdmin):
+    list_display = ('name', 'is_active')
+    list_filter = ('is_active',)
```

Comment:
Also set `view_on_site = False` because I know someone will try to follow that link if it's present